packages/components/src/__future__/Tabs/constants.ts

```diff
@@ -0,0 +1 @@
+export const SCROLL_AMOUNT = 120
```

Comment:
Nit: Haha, probably didn't need to make a separate file for this since we only use it in the one place. But no matter.